Makefile

```diff
@@ -19,7 +19,7 @@ debug:
 nocairo:
 	$(GO) build -ldflags '-X main.BuildVersion=$(VERSION)' $(PKG_CARBONAPI)
 
-carbonzipper:
+carbonzipper: $(shell find . -name '*.go' | grep -v 'vendor')
```

Comment:
Same question as in one of the previous commits - I don't understand the reason behind this change.